sandboxes/heka/output/heka_s3_partition.lua

```diff
@@ -0,0 +1,281 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+require "cjson"
+require "io"
+require "os"
+require "string"
+require "table"
+
+files               = {}
+local fh_cnt        = 0
+local time_t        = 0
+local buffer_cnt    = 0
+
+local hostname              = read_config("Hostname")
+local batch_path            = read_config("batch_path") or error("batch_path must be specified")
+local s3_path               = read_config("s3_path") or error("s3_path must be specified")
+local max_file_handles      = read_config("max_file_handles") or 1000
+local max_file_size         = read_config("max_file_size") or 1024 * 1024 * 500
+local max_file_age          = read_config("max_file_age") or 60 * 60
+local flush_on_shutdown     = read_config("flush_on_shutdown")
```

Comment:
Never mind, I see this is explicitly mentioned below. Fine with me to leave it defaulting to false.